src/Plugin/VoteUpDownWidgetInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\vud\Plugin;
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+
+/**
+ * Defines an interface for Vote Up/Down Widget plugins.
+ */
+interface VoteUpDownWidgetInterface extends PluginInspectionInterface {
+
+  /**
+   * Method to return the label of the plugin whenever required
+   *
+   * @return mixed
+   */
+  public function get_label();
+
+  /**
+   * Method to alter template variables on render
+   * @param $widget_template
+   * @param $variables
+   *
+   * @return mixed
+   */
+  public function alter_template_variables($widget_template, $variables);
```

Comment:
Still pending, you changed the base class implementation name, but not the one in the interface.